### DISCUSSION TITLE
Actions Support

### DISCUSSION THREAD

**Discussion by wangyucui1002926** [community/community#80881] [Actions]
### Select Topic Area

Product Feedback

### Body

Does GitHub's official supporter support Chinese?
About Standard support type,Is it include  phone support or only web support? is there a SLA?

**Comment by Akash1134** [comment]
Hi @wangyucui1002926 ,

You can navigate the [GitHub Actions Support](https://support.github.com/features/actions) site to get help on your issue related to Actions. 

Hope that helps!
